src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+### Allow Okteto Build service privileged access
+
+By default, running privileged containers in OpenShift is not allowed. This restriction impacts the installation of the Okteto Build. To give privileged access to the Okteto Build service, run the following command:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+### Allow the Okteto Daemon service to access the host
+
+By default, access to the host in OpenShift is not allowed. This restriction impacts the installation of the Okteto Daemon service. To give access to the host to the Okteto Daemon run the following command:
+
+```bash
+oc adm policy add-scc-to-user hostaccess -z okteto -n okteto
+```
+
+### Copy OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `apps.<your-openshift-subdomain>` . We recommend reusing it when installing Okteto.
+
+To do that, copy the certificate to the `okteto` namespace (replace `<cluster-name>` with your cluster name):
+
+```yaml
+kubectl get secret <clustername>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
+```
+
+### Get the OpenShift Ingress Controller IP
+
+We recommend installing Okteto behind the Openshift Ingress controller. To do this, remember the IP of your Openshift Ingress Controller:
+
+```bash
+kubectl get svc router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+We will refer to this value using `<ingress-controller-ip>`.
+
+:::warning
+Installing Okteto behing the OpenShift Ingress controller means the following features aren't available:
+- [Auto wake on incoming requests](admin/cleanup.mdx#inactive-applications-and-resources)
+- [Private Endpoints](core/endpoints/private-endpoints.mdx)
+:::
+
+## Installing Okteto
+
+Okteto is installed using a Helm chart. Let's start the process:
+
+### Add the Okteto Helm repository
+
+You'll need to add the Okteto Helm repository to be able to install Okteto:
+
+```bash
+helm repo add okteto https://charts.okteto.com
+helm repo update
+```
+
+### Create the Helm configuration file
+
+In order to install Okteto you need to first create a `config.yaml` for the installation process.
+
+Create the following Helm configuration file for Okteto in, for example, a file called `config.yaml`.
+Remember to replace `<<your-okteto-license>>`, `<your-openshift-subdomain>`, `<cluster-name>` and `<ingress-controller-ip>` with your own values:
+
+```yaml
+license: "REPLACE ME WITH YOUR OKTETO LICENSE"
+
+openshift: # supports Openshift Routes as Okteto Endpoints
```

Comment:
What are Okteto Endpoints? I think this comment can confuse the reader instead of helping him understand what he's doing (I know it's a tough balance)

Reply:
removed